formulas/redis/functions/redis-set-config-line

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+#
+# Public: Sets a value in the redis.conf file.  If the config already
+# has the configKey provided it will overwrite it.  If the configValue is
+# empty, the line which starts with the configKey will be removed.
+#
+# $1 - The name of the config value you would like to set.
+# $2 - The value you would like to set the config value to.
+# If the value is empty it will remove that config setting.
+#
+# Examples
+#
+#   redisSetConfigLine "requiresauth" "my password"
+#
+#   redisSetConfigLine "unsetthisvalue" ""
+#
+# Returns nothing.
+redisSetConfigLine() {
+    wickGetArgument configKey 0 "$@"
+    wickGetArgument configValue 1 "$@"
+    redisConfPath="/etc/redis.conf"
+    if [[ -z "$configValue" ]]; then
+        wickInfo "Unsetting $configKey in $redisConfPath"
+        sed -i "/^$configKey/d" "$redisConfPath"
```

Comment:
Why not use something like this?

    wickSetConfigLine "$redisConfPath" "" "$configKey"